components/director/pkg/graphql/graphqlizer/fields.go

```diff
@@ -415,16 +415,25 @@ func (fp *GqlFieldsProvider) ForLabel() string {
 }
 
 // ForRuntime missing godoc
-func (fp *GqlFieldsProvider) ForRuntime() string {
-	return fmt.Sprintf(`
+func (fp *GqlFieldsProvider) ForRuntime(ctx ...FieldCtx) string {
+	return addFieldsFromContext(fmt.Sprintf(`
 		id
 		name
 		description
 		labels 
 		status {condition timestamp}
 		metadata { creationTimestamp }
 		auths {%s}
-		eventingConfiguration { defaultURL }`, fp.ForSystemAuth())
+		runtimeContexts {%s}
+		eventingConfiguration { defaultURL }`, fp.ForSystemAuth(), fp.Page(fp.ForRuntimeContext())), ctx, []string{"Runtime.runtimeContext"})
+}
+
+// ForRuntimeContext missing godoc
+func (fp *GqlFieldsProvider) ForRuntimeContext() string {
+	return `id
+			key
+			value
+			labels`
```

Comment:
The runtimeContexts still have labels. We have removed the labels from RuntimeContextInput so that the customers can not provide labels when creating or updating their RuntimeContexts. However we will be labeling the RuntimeContexts with the scenario labels and we will return them when RuntimeContexts are listed.